admin/client/src/legacy/ModelAdmin.js

```diff
@@ -3,6 +3,8 @@
  */
```

Comment:
Good point, I've added it to `legacy.js` now and verified that the "expand csv stuff" feature works in ModelAdmin (so the JS kicks in)